README.md

```diff
@@ -835,6 +838,10 @@ Some detail on how this works:
     investors get their money back and also participate in proceeds from common stock.
   - Whether there is a **cap**, which limits the payout if it is participating.
 
+  For more information on each of these,
+  [this Medium post](https://medium.com/@CharlesYu/the-ultimate-guide-to-liquidation-preferences-478dda9f9332)
```

Comment:
Nit: Omit Medium.

Reply:
Formatting: Best not to put content both before and after a sub-bulleted section.